MatchingGame.js

```diff
@@ -59,14 +57,14 @@ const cardGenerator = () => {
         });
     });
 };
-//Check card
+// Check card
 const checkCards = (e) => {
     console.log(e);
     const clickedCard = e.target;
     clickedCard.classList.add("flipped");
     const flippedCards = document.querySelectorAll('.flipped');
     const toggleCard = document.querySelectorAll('.toggleCard');
-    //Logic
```

Comment:
No point in a comment that just says "Logic", you can elaborate in what the logic is or if you feel the logic is obvious no need for a comment

In general, I recommend putting the majority of the info about the function in a comment above the function (where you wrote `Check card`) and if there is something unintuitive/unusual happening in the function itself write an additional comment explaining it. There is a such things as "over commenting"